src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobResumeBuild.java

```diff
@@ -0,0 +1,45 @@
+package com.tikal.jenkins.plugins.multijob;
+
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.Cause;
+import hudson.model.ProminentProjectAction;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import java.io.IOException;
+
+public class MultiJobResumeBuild implements ProminentProjectAction {
+
+	private final AbstractBuild<?, ?> build;
+
+	public MultiJobResumeBuild(AbstractBuild<?, ?> build) {
```

Comment:
It's better to use <code>Run</code>s instead of <code>AbstractBuild</code>s in the new code. Particular run types (e.g. <code>WorkflowRun</code> from Workflow plugin) do not inherit <code>AbstractBuild</code>